packages/wrangler/src/user.tsx

```diff
@@ -963,6 +979,8 @@ export async function login(props?: LoginProps): Promise<boolean> {
     server.listen(8976);
   });
 
+  await openInBrowser(urlToOpen);
```

Comment:
NOTE: moving this here, helps to ensure that the HTTP server that will receive the callback from the OAuth provider is already listening before we try to open the OAuth web page.